js/tak.js

```diff
@@ -0,0 +1,290 @@
+function alert(type, msg) {
+    $('#alert-text').text(msg);
+    var $alert = $('#alert');
+    $alert.removeClass("alert-success alert-info alert-warning alert-danger");
+
+    $alert.addClass("alert-"+type);
+    $alert.removeClass('hidden');
+    $alert.stop(true, true);
+    $alert.fadeTo(4000, 500).slideUp(500, function() {
+        $alert.addClass('hidden');
+    });
+    alert2(type, msg);
+}
+function alert2(type, msg) {
+    $('#alert-text2').text(msg);
+    var $alert = $('#alert2');
+    $alert.removeClass("alert-success alert-info alert-warning alert-danger");
+
+    $alert.addClass("alert-"+type);
+    $alert.removeClass('hidden');
+    $alert.stop(true, true);
+    $alert.fadeTo(4000, 500).slideUp(500, function() {
+        $alert.addClass('hidden');
+    });
+}
+
+var camera, scene, renderer, light, canvas, controls = null;
+var raycaster = new THREE.Raycaster();
+var highlighter;
+var mouse = new THREE.Vector2();
+var offset = new THREE.Vector3();
+
+// settings
+var antialiasing_mode = false;
+var stack_dist = 105;
```

Comment:
everything from //settings to //end settings  (except antialiasing_mode) - those variables are to control board settings, so please put them in board.js. antialiasing_mode could be left here.